pkg/apis/velero/v1/backup_repository_types.go

```diff
@@ -51,9 +51,6 @@ const (
 	BackupRepositoryPhaseNew      BackupRepositoryPhase = "New"
 	BackupRepositoryPhaseReady    BackupRepositoryPhase = "Ready"
 	BackupRepositoryPhaseNotReady BackupRepositoryPhase = "NotReady"
-
-	BackupRepositoryTypeRestic string = "restic"
```

Comment:
I don't have a very strong opinion but why is this needed?

Reply:
If you mean the variable, it is used in these places https://github.com/search?q=repo%3Avmware-tanzu%2Fvelero%20BackupRepositoryTypeRestic&type=code

If you mean the change to another file, is just for semantics (it is a constant, not a type used to generate the CRD, so lets move to constants file)